Patch-Source: https://github.com/tectonic-typesetting/tectonic/pull/1202/commits/d260961426b01f7643ba0f35f493bdb671eeaf3f

Fix `'auto' parameter not permitted in this context` error when building
against icu 76.1.
---
From d260961426b01f7643ba0f35f493bdb671eeaf3f Mon Sep 17 00:00:00 2001
From: Winston H <56998716+winstxnhdw@users.noreply.github.com>
Date: Wed, 26 Jun 2024 07:05:10 +0800
Subject: [PATCH] fix: use c++ 17

---
 crates/xetex_layout/build.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crates/xetex_layout/build.rs b/crates/xetex_layout/build.rs
index bdd0d03fc..558fd29e1 100644
--- a/crates/xetex_layout/build.rs
+++ b/crates/xetex_layout/build.rs
@@ -86,7 +86,7 @@ fn main() {
     let mut cppcfg = cc::Build::new();
 
     let cppflags = [
-        "-std=c++14",
+        "-std=c++17",
         "-Wall",
         "-Wdate-time",
         "-Wendif-labels",
